src/App.tsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+
+import { Todo } from './types/Todo';
+
+import { TodoList } from './components/TodoList';
+
+const preparedTodos = todos.map((todo) => {
+  const todoCopy: Todo = {
+    ...todo,
+    user: users.find(user => user.id === todo.userId) || null,
+  };
+
+  return todoCopy;
+});
```

Comment:
Could write it without return.
```suggestion
const preparedTodos = todos.map((todo) => ({
    ...todo,
    user: users.find(user => user.id === todo.userId) || null,
});
```